### PR TITLE
Making sure documentation is ready for GA

### PR DESCRIPTION
**Pull request by Abuelodelanada**
## Issue

Make sure the documentation is ready for GA

## Context

Going to GA!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Overview
 
 This documents explains the processes and practices recommended for
-contributing enhancements to the Prometheus charm.
+contributing enhancements to the Prometheus Charmed Operator.
```
Comment (sed-i) [inline CONTRIBUTING.md:6]: At first glance it seems that "charmed operator" should not be capitalized here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,51 +12,64 @@ contributing enhancements to the Prometheus charm.
...
 - Familiarising yourself with the
   [Charmed Operator Framework](https://juju.is/docs/sdk)
   library will help you a lot when working on new features or bug fixes.
 - All enhancements require review before being merged. Code review
   typically examines
   + code quality
   + test coverage
-  + user experience for Juju administrators
-  this charm.
+  + user experience for Juju administrators.
```
Comment (sed-i) [inline CONTRIBUTING.md:22]: Do you think the phrase "Juju administrators" could be misunderstood?
If yes, how about:

```suggestion
  + cluster admin ux
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -12,51 +12,64 @@ contributing enhancements to the Prometheus charm.
...
-```
+A typical development setup for charms created with the [Charmed Operator Framework](https://juju.is/docs/sdk) contains:
+
+
+- [Charmcraft](https://github.com/canonical/charmcraft) - Developer tooling for creating, building and publishing Charmed Operators
+- [Juju](https://juju.is/) - a Charmed Operator Lifecycle Manager (OLM), used for deploying and managing operators
+- [Multipass](https://multipass.run/) - a lightweight Ubuntu virtual machine manager (optional)
+- [MicroK8s](https://microk8s.io/) - a low-ops Kubernetes distribution we’ll use for testing our Charmed Operator (optional if developing a Charmed Operator for Kubernetes)
+
+Please [follow this guide](https://juju.is/docs/sdk/dev-setup) which will walk through the installation of these tools to get you started with charm development.
```
Comment (sed-i) [inline CONTRIBUTING.md:37]: We should probably offload as much as sensible to the dev-setup doc.
How about:

```suggestion
This prometheus operator is developed with the [Charmed Operator Framework](https://juju.is/docs/sdk).

Please follow the [dev-env setup guide](https://juju.is/docs/sdk/dev-setup) to get started.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -65,36 +78,22 @@ $ juju deploy --trust \
...
-[harness](https://ops.readthedocs.io/en/latest/#module-ops.testing). These
-tests may executed by doing
-
-```bash
-$ tox -e unit
-```
-
-It is expected that unit tests should provide at least 80% code coverage.
 
 ## Code Overview
```
Comment (sed-i) [inline CONTRIBUTING.md:83]: I wounder if we should entirely drop the "code overview" section.

File: INTEGRATING.md [hunk 5]
```diff
@@ -31,11 +31,10 @@ Any time new relations are made, or existing ones changed, between the
...
+2. [Grafana Charmed Operator](https://charmhub.io/grafana-k8s), over the `grafana_datasource` interface, to provide a dashboard for viewing metrics aggregated by Prometheus. These dashboards may be customised by charms that relate to Grafana.
 
-3. Prometheus forwards alerts to one or more
-[Alertmanagers](https://charmhub.io/alertmanager-k8s) that are related
-to it.
+3. [Alertmanager Charmed Operator](https://charmhub.io/alertmanager-k8s), over the `alertmanager` interface, so Prometheus may forward alerts to it.
+
+4. [Traefik Charmed Operator](https://charmhub.io/traefik-k8s), over the `ingress_per_unit` interface, so Prometheus may be reached from outside the Kubernetes cluster it is running on.
+
+5. [Catalogue CharmedOperator](https://github.com/canonical/catalogue-k8s-operator/tree/main/charm), over the `catalogue` interface, so Prometheus can be published in the service catalogue web page.
```
Comment (sed-i) [inline INTEGRATING.md:40]:
```suggestion
5. [Catalogue Charmed Operator](https://charmhub.io/catalogue-k8s), over the `catalogue` interface, so Prometheus can be published in the service catalogue web page.
```

**Review by dstathis — APPROVED** [review]

File: CONTRIBUTING.md (hunk 4 above)
Comment (Abuelodelanada) [reply CONTRIBUTING.md:77]: yes, it seems to me that it does not add much value...

**Review by sed-i — APPROVED** [review]

**Pull request merged**